backend/api/v1/auth.py

```
@@ -29,160 +43,140 @@ class UserProfile(BaseModel):
 async def logout(current_user: Dict[str, Any] = Depends(get_current_user)):
     """
     Logout the current user by invalidating their session.
+    Note: With JWT tokens, true logout requires token blacklisting on client side.
     """
     try:
-        supabase_client.auth.sign_out()
+        # Sign out from Supabase (this invalidates the refresh token)
+        supabase_anon.auth.sign_out()
         return {"message": "Successfully logged out"}
     
     except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)